source/hardware/connecting-to-the-serial-console.rst

```diff
@@ -93,6 +93,10 @@ port, start PuTTY and then set the following parameters:
 The session may be saved for later use. Click **Open** to start the
 serial console session.
 
+MINICOM
+~~~~~
+`minicom -D /dev/ttyUSB0 -R 115200`
```

Comment:
This should not be surrounded by single backticks (`` ` ``), it should be formatted in a block. This page hasn't been updated with the current recommended syntax, but it should be:

```
.. code-block:: console

  minicom -D /dev/ttyUSB0 -R 115200
```

Note the blank lines and spacing.

Reply:
Ok thanks you for the comment, i've fix that :) 
Sorry for the mistake